docs/tutorials/using-lego-dimensions-toy-pad.md

```diff
@@ -0,0 +1,363 @@
+associated to each reader.
+
+## Credits
+
+Since the beginning of 2015 several people have been reverse engineering the
+LEGO Toy Pad.
+This tutorial is strongly based on **woodenphone** [work](https://github.com/woodenphone/lego_dimensions_protocol) available at GitHub.
+
+
+## Requirements
+
+You will need:
+
+* a proper LEGO Dimensions Toy Pad
+* a Mindstorms EV3
+* an available USB port
+* python and pysub
+ 
+Several people reported a difference between PS3/PS4/Wii devices and Xbox so
+this tutorial most probaly will not work with the Xbox type.
+
+You don't really need a Mindstorms EV3 as this tutorial can be used on almost any
+recent linux system (like my Ubuntu laptop or my Raspberry Pi). If using ev3dev,
+just be sure to use a recent version - this tutorial was tested with kernel
+`4.4.15-13-ev3dev-ev3`.
+
+Of course, you need an available USB port so if you're already using the Mindstorms
+EV3 with an USB Wi-Fi dongle you will also need an USB Hub.
+
+After you connect the LEGO Toy Pad it should be recognized as an HID device:
+
+    robot@ev3dev:~# dmesg
+    ...
+    usb 2-1.2: new full-speed USB device number 8 using ehci-pci
+    usb 2-1.2: New USB device found, idVendor=0e6f, idProduct=0241
+    usb 2-1.2: New USB device strings: Mfr=1, Product=2, SerialNumber=3
+    usb 2-1.2: Product: LEGO READER V2.10
+    usb 2-1.2: Manufacturer: PDP LIMITED. 
+    usb 2-1.2: SerialNumber: P.D.P.000000
+    hid-generic 0003:0E6F:0241.0006: hiddev0,hidraw3: USB HID v1.00 Device
+    [PDP LIMITED.  LEGO READER V2.10] on usb-0000:00:1d.0-1.2/input0
+
+You can also check with lsusb:
+
+    robot@ev3dev:~# lsusb
+    ...
+    Bus 002 Device 008: ID 0e6f:0241 Logic3
+    ...
+
+
+You also need **python** and **pyusb**. Most linux distributions already
+include python as default so you probably only need to install the pyusb library
+with:
+
+    sudo pip install pyusb
+
+but you may need to install pip first:
+
+    sudo apt-get update
+    sudo apt-get install python-pip
+
+
+
+## Initializing
+
+The following script will check for the presence of a LEGO Toy Pad and initializes
+it, turning the middle (round) pad red for one second. It should be invoked with
+`sudo` or root privileges.
+
+{% highlight python %}
+
+#!/usr/bin/python
+
+import usb.core
+import usb.util
+from time import sleep
+
+TOYPAD_INIT = [0x55, 0x0f, 0xb0, 0x01, 0x28, 0x63, 0x29, 0x20, 0x4c, 0x45, 0x47, 0x4f, 0x20, 0x32, 0x30, 0x31, 0x34, 0xf7, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00]
+
+PAD1_RED = [0x55, 0x0e, 0xc8, 0x06, 0x01, 0xff, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00, 51, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00]
+
+PADS_OFF = [0x55, 0x06, 0xc0, 0x02, 0x00, 0x00, 0x00, 0x00, 29, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00]
+
+def init_usb():
```

Comment:
Your indentations are inconsistent here. The preferred python style is to indent using 4 spaces.